modules/drivers/velodyne/dag/velodyne_64.dag

```diff
@@ -0,0 +1,177 @@
+# Define all coms in DAG streaming.
+
+##################################################
+#                   drivers                      #
+##################################################
+
+module_config {
+    module_library : "/apollo/bazel-bin/modules/drivers/velodyne/driver/libvelodyne_driver_component.so"
+
+    # 64
+    components {
+      class_name : "VelodyneDriverComponent"
+      config {
+        name : "velodyne_64_driver"
+        config_file_path : "/apollo/modules/drivers/velodyne/conf/velodyne64_conf.pb.txt"
+      }
+    }
+    # 16_front_up
+    components {
+      class_name : "VelodyneDriverComponent"
+      config {
+        name : "velodyne_16_front_up_driver"
+        config_file_path : "/apollo/modules/drivers/velodyne/conf/velodyne16_front_up_conf.pb.txt"
+      }
+    }
+    # 16_front_center
+    components {
```

Comment:
Looks like this is more than just 64 beam?

Reply:
Yeah. Let's rename to velodyne_64_16.dag

Reply:
> Yeah. Let's rename to velodyne_64_16.dag

OK